From f2818732f71302d3b98e169860536c12ee3a3301 Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Mon, 9 Jul 2012 12:58:16 +0800
Subject: ENGR00216013-3 vpu: add VPU_IOC_PHYMEM_CHECK ioctl.

Add VPU_IOC_PHYMEM_CHECK ioctl in header file.
This IOCTL will check the phy memory address is valid or not.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mxc_vpu.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/plat-mxc/include/mach/mxc_vpu.h b/arch/arm/plat-mxc/include/mach/mxc_vpu.h
index da4f39d..0c0fa2a 100755
--- a/arch/arm/plat-mxc/include/mach/mxc_vpu.h
+++ b/arch/arm/plat-mxc/include/mach/mxc_vpu.h
@@ -57,6 +57,7 @@ struct vpu_mem_desc {
 #define VPU_IOC_GET_SHARE_MEM   _IO(VPU_IOC_MAGIC, 12)
 #define VPU_IOC_QUERY_BITWORK_MEM  _IO(VPU_IOC_MAGIC, 13)
 #define VPU_IOC_SET_BITWORK_MEM    _IO(VPU_IOC_MAGIC, 14)
+#define VPU_IOC_PHYMEM_CHECK	_IO(VPU_IOC_MAGIC, 15)
 
 #define BIT_CODE_RUN			0x000
 #define BIT_CODE_DOWN			0x004
-- 
1.7.9.5

